backend/reviving_ontology_api_stuff.py

```diff
@@ -1,9 +1,16 @@
-"""TermHub backend
+"""Temporary routes for trying stuff out. will merge into backend.py
```

Comment:
The idea is that we will delete this file and merge anything useful into `backend.py`, I believe